aci/resource_aci_fvctx.go

```diff
@@ -213,6 +225,8 @@ func setVRFAttributes(fvCtx *models.VRF, d *schema.ResourceData) (*schema.Resour
 	d.Set("name", fvCtxMap["name"])
 
 	d.Set("annotation", fvCtxMap["annotation"])
+	d.Set("scope", fvCtxMap["scope"])
```

Comment:
did you test adding/updating these values?

Reply:
Yes, I did

Reply:
How did you test this behaviour? 

I think this only reads the data from apic and does not set it in the payload.

I would expect you to set it [here](https://github.com/CiscoDevNet/terraform-provider-aci/pull/1238/files#diff-0baf6466d84e03887920db87e9afed21095c499dd9a4c58bc6cff26d77dc05f1R274-R298) and that fvCtxAttr it is passed into the `models.NewVRF(fmt.Sprintf("ctx-%s", name), TenantDn, desc, fvCtxAttr)` function with the scope and pc_tag attribute.

The same would apply for the update.

Also I do not see update of the aci-go-client vendor files.

Reply:
create and update not required for these attributes